content/1.7/back-office/design/image-settings/image-settings-list.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Image Settings
+weight: 1
+---
+# SPECIFICATIONS - DESIGN > IMAGE SETTINGS
+
+## Image Settings listing
+
+![Image Settings listing](static/img/image-settings-image-settings.png)
+
+During the first Prestashop installation process on the server, Prestashop installs 7 default image settings with the following default properties.
+- **cart_default** - Products is set to enabled, Categories is set to disabled, Brand is set to disabled, Suppliers is set to disabled, Stores is set to disabled.
+- **small_default** - Products is set to enabled, Categories is set to enabled, Brand is set to enabled, Suppliers is set to enabled, Stores is set to disabled.
+- **medium_default** - Products is set to enabled, Categories is set to disabled, Brand is set to enabled, Suppliers is set to enabled, Stores is set to disabled.
+- **home_default** - Products is set to enabled, Categories is set to disabled, Brand is set to disabled, Suppliers is set to disabled, Stores is set to disabled.
+- **large_default** - Products is set to enabled, Categories is set to disabled, Brand is set to enabled, Suppliers is set to enabled, Stores is set to disabled.
+- **category_default** - Products is set to disabled, Categories is set to enabled, Brand is set to disabled, Suppliers is set to disabled, Stores is set to disabled.
+- **stores_default** - Products is set to disabled, Categories is set to disabled, Brand is set to disabled, Suppliers is set to disabled, Stores is set to enabled.
+
+## Image Settings table properties
+
+- **Checkboxes** - each image setting can be checkboxed separately, if there is a need of Bulk Actions executions with certain image setting. By default, there is nothing selected.
+- **ID** - specific digital value to describe the image setting unique operator. By default they are arranged automatically.
+- **Name** - specific image setting value names. By default they are installed automatically.
+- **Width** - specific image setting value, describing the width of the image in pixels. By default the specific width is installed.
+- **Height** - specific image setting value, describing the height of the image in pixels. By default the specific height is installed.
+- **Products** - specific image setting value, indicating the enabled checkbox or the disabled cross operator, which can be clickable. Clicking these operators can toggle the enabling and disabling functionality directly from listing page.
+- **Categories** - specific image setting value, indicating the enabled checkbox or the disabled cross operator, which can be clickable. Clicking these operators can toggle the enabling and disabling functionality directly from listing page.
```

Comment:
It crashes because PS doesn't have a value for the categories, maybe. Just to be sure, does it crash if you add a second non default value enable and the rest are disabled ?
If so, we can add a warning above the list, saying like "if for all the image type are disabled for a categories of images than the PS will crash"

Reply:
The FO homepage crashes, with the error with the meaning of "home_default":
<img width="329" alt="image" src="https://user-images.githubusercontent.com/29095235/150326218-0ce75a1a-78da-4ccd-bb59-0bf2f56a6029.png">

I guess, it comes from the "home_default" image setting. When I enable, for example, setting for Products only, exception dissapears.

Reply:
> The FO homepage crashes, with the error with the meaning of "home_default": <img alt="image" width="329" src="https://user-images.githubusercontent.com/29095235/150326218-0ce75a1a-78da-4ccd-bb59-0bf2f56a6029.png">
> 
> I guess, it comes from the "home_default" image setting. When I enable, for example, setting for Products only, exception dissapears.

What was the configuration ?

Reply:
@MatShir 
Adding some more information: 
Exact crash in ImageRetriever.php file:
![image](https://user-images.githubusercontent.com/29095235/150748583-a4c8292b-b0cd-4086-86f0-69634bdc9eb7.png)

The configuration:
![image](https://user-images.githubusercontent.com/29095235/150748752-3cc5ba2c-14c1-43db-b64f-a8078730806b.png)

Reply:
@MatShir not quite sure, should I raise it as a bug in community, or update the specs with notifications "if for all the image type are disabled for a category of images than the PS will crash" something like that? :)